### PR TITLE
Development

### PR DESCRIPTION
**Pull request by chiararelandini**
**Hello world example service** exposes the following endpoints:
- `/hello`: hello world endpoint
- `/hello/{name}`: hello {name} endpoint
- `/-/metrics`: metrics
- `/-/healthz`: kubernetes health probe
- `/-/ready`: kubernetes readiness probe

### CODE REVIEW

File: .helidon [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
...chiara/.m2/repository/io/helidon/metrics/helidon-metrics/2.2.1/helidon-metrics-2.2.1.jar,/home/chiara/.m2/repository/io/helidon/config/helidon-config-mp/2.2.1/helidon-config-mp-2.2.1.jar,/home/chiara/.m2/repository/io/helidon/common/helidon-common-service-loader/2.2.1/helidon-common-service-loader-2.2.1.jar,/home/chiara/.m2/repository/org/eclipse/microprofile/config/microprofile-config-api/1.4/microprofile-config-api-1.4.jar,/home/chiara/.m2/repository/org/eclipse/microprofile/metrics/microprofile-metrics-api/2.3.2/microprofile-metrics-api-2.3.2.jar,/home/chiara/.m2/repository/io/helidon/common/helidon-common-context/2.2.1/helidon-common-context-2.2.1.jar,/home/chiara/.m2/repository/io/netty/netty-buffer/4.1.58.Final/netty-buffer-4.1.58.Final.jar,/home/chiara/.m2/repository/com/fasterxml/jackson/datatype/jackson-datatype-jsr310/2.12.1/jackson-datatype-jsr310-2.12.1.jar,/home/chiara/.m2/repository/com/fasterxml/jackson/core/jackson-annotations/2.12.1/jackson-annotations-2.12.1.jar,/home/chiara/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.12.1/jackson-core-2.12.1.jar,/home/chiara/.m2/repository/com/fasterxml/jackson/core/jackson-databind/2.12.1/jackson-databind-2.12.1.jar
```
Comment (fredmaggiowski) [inline .helidon]: this file contains local paths related to your system, maybe it's better not to add it in the vcs?

File: .dockerignore [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.git
+.vscode
```
Comment (ilteoood) [inline .dockerignore:2]: I would also insert the `.idea` folder

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,41 @@
+
```
Comment (fredmaggiowski) [inline Dockerfile]: can you please remove this useless empty line?

File: .github/workflows/maven.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ master ]
```
Comment (ilteoood) [inline .github/workflows/maven.yml]: are you sure about this? You only have the `main` branch

File: .github/workflows/maven.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  pull_request:
+    branches: [ master ]
```
Comment (ilteoood) [inline .github/workflows/maven.yml]: same here

File: .gitignore [hunk 6]
```diff
@@ -0,0 +1,5 @@
+.vscode
+.classpath
+
```
Comment (ilteoood) [inline .gitignore:6]: I would also insert the `.idea` folder

File: .settings/org.eclipse.core.resources.prefs [hunk 7]
```diff
@@ -0,0 +1,5 @@
+eclipse.preferences.version=1
```
Comment (ilteoood) [inline .settings/org.eclipse.core.resources.prefs]: I think that `.settings` folder should be placed inside `.gitignore` and `dockerignore`, and removed from vcs

File: .settings/org.eclipse.jdt.apt.core.prefs [hunk 8]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```
Comment (ilteoood) [inline .settings/org.eclipse.jdt.apt.core.prefs]: I think that .settings folder should be placed inside .gitignore and dockerignore, and removed from vcs

File: .settings/org.eclipse.jdt.core.prefs [hunk 9]
```diff
@@ -0,0 +1,9 @@
+eclipse.preferences.version=1
```
Comment (ilteoood) [inline .settings/org.eclipse.jdt.core.prefs]: I think that .settings folder should be placed inside .gitignore and dockerignore, and removed from vcs

File: .settings/org.eclipse.m2e.core.prefs [hunk 10]
```diff
@@ -0,0 +1,4 @@
+activeProfiles=
```
Comment (ilteoood) [inline .settings/org.eclipse.m2e.core.prefs]: I think that .settings folder should be placed inside .gitignore and dockerignore, and removed from vcs

**Review by ilteoood — CHANGES_REQUESTED** [review]

**Review by ilteoood — APPROVED** [review]

File: Dockerfile.native [hunk 11]
```diff
@@ -0,0 +1,42 @@
+
```
Comment (fredmaggiowski) [inline Dockerfile.native]: same here, this line is unnecessary

**Review by fredmaggiowski — CHANGES_REQUESTED** [review]
some minor changes, then it's fine!

**Review by fredmaggiowski — APPROVED** [review]

**Pull request merged**